src/Framework/Microsoft.AspNetCore.App/Microsoft.AspNetCore.App.props

```diff
@@ -45,13 +39,9 @@
     <Dependency Include="Microsoft.AspNetCore.Http"                                    Version="$(MicrosoftAspNetCoreHttpPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.HttpOverrides"                           Version="$(MicrosoftAspNetCoreHttpOverridesPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.HttpsPolicy"                             Version="$(MicrosoftAspNetCoreHttpsPolicyPackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore"            Version="$(MicrosoftAspNetCoreIdentityEntityFrameworkCorePackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.Identity.UI"                             Version="$(MicrosoftAspNetCoreIdentityUIPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Identity"                                Version="$(MicrosoftAspNetCoreIdentityPackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.JsonPatch"                               Version="$(MicrosoftAspNetCoreJsonPatchPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Localization.Routing"                    Version="$(MicrosoftAspNetCoreLocalizationRoutingPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Localization"                            Version="$(MicrosoftAspNetCoreLocalizationPackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.MiddlewareAnalysis"                      Version="$(MicrosoftAspNetCoreMiddlewareAnalysisPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc.Abstractions"                        Version="$(MicrosoftAspNetCoreMvcAbstractionsPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc.Analyzers"                           Version="$(MicrosoftAspNetCoreMvcAnalyzersPackageVersion)" />
```

Comment:
Analyzers go out of shared fx, don't they?

Reply:
The analyzers were never in the shared fx, but they were in the metapackage. I'm not attempting to make the metapackage into a flat package in this PR. I'd like to address issues with packages that deliver build and analyzer assets in a separate PR. (See https://github.com/aspnet/AspNetCore/issues/3611)